### PR TITLE
Check `webkit` prefix format for api/css compat data.

### PR DESCRIPTION
**Pull request by johngian**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [x] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
(Related to #3670)

Thanks for the PR on this!  Overall, it's looking nice, however I think that we can generalize it for "moz" and "ms" as well ("o" for pre-Blink Opera may get a little difficult).  It's not particularly required, but it's bonus points if you can add said functionality in.  :wink:

Oh and welcome to BCD!

**Comment by johngian** [comment]
Hey @vinyldarkscratch thanks for the review. I updated the PR with the rest of the prefixes.

File: test/lint.js [hunk 1]
```diff
@@ -65,13 +67,19 @@ function load(...files) {
...
           hasSyntaxErrors = true;
           console.error(e);
         }
-        if (hasSyntaxErrors || hasSchemaErrors || hasStyleErrors || hasBrowserErrors || hasVersionErrors) {
-          hasErrors = true;
+
+        hasErrors = [
+          hasSyntaxErrors, hasSchemaErrors, hasStyleErrors,
+          hasBrowserErrors, hasVersionErrors, hasApiErrors
+        ].some(x => x === true);
```
Comment (ExE-Boss) [inline test/lint.js]: This won’t work because `hasErrors` is globally persistent to the process.

File: test/test-prefix.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+path = require('path');
+
+function checkPrefix(data, category, errors, prefix, path="") {
+  for (var key in data) {
+    if (key === "prefix" && typeof(data[key]) === "string") {
+      if (data[key].includes(prefix)) {
+        var error = `${prefix} prefix is wrong for key: ${path}`;
+        var rules = [
+          category == "api" && data[key].startsWith(prefix),
```
Comment (ExE-Boss) [inline test/test-prefix.js]: For API, the prefix can be mixed case:
<dl>
<dt>Mozilla Firefox</dt>
<dd><code>moz</code>, <code>Moz</code></dd>
<dt>Chrome and Safari</dt>
<dd><code>webkit</code>, <code>WebKit</code> (and maybe even <code>Webkit</code> or <code>webKit</code>)</dd>
<dt>IE/Edge</dt>
<dd><code>ms</code>, <code>MS</code></dd>
</dl>

Comment (jpmedley) [reply test/test-prefix.js]: FWIW, `Webkit` and `webKit` do not currently appear in Chrome's code base.

**Comment by johngian** [comment]
PR updated

**Pull request merged**